src/processor/mod.rs

```diff
@@ -1,2 +1,4 @@
+#[cfg(feature = "pyo3")]
 pub mod model;
+#[cfg(feature = "pyo3")]
 pub mod py;
```

Comment:
If there's nothing else included from here, would it work the same to put the cfg line on the `mod processor` in lib.rs?

Reply:
👀